packages/dapi-grpc/clients/platform/v0/nodejs/platform_protoc.js

```diff
@@ -10055,7 +10055,8 @@ proto.org.dash.platform.dapi.v0.GetProofsRequest.GetProofsRequestV0.IdentityRequ
 proto.org.dash.platform.dapi.v0.GetProofsRequest.GetProofsRequestV0.IdentityRequest.Type = {
   FULL_IDENTITY: 0,
   BALANCE: 1,
-  KEYS: 2
+  KEYS: 2,
+  REVISION: 3
```

Comment:
I thought we were asking for balance and revision.

Reply:
yes, in the section where you left the next comment we're asking for both

Reply:
I don't understand your reply.

Reply:
> I thought we were asking for balance and revision.

we do, balance for identity by id + revision for identity by id, no need for (balance + revision) for identity by id because the former works and composable

Reply:
Got it, multiple requests at the identity level with the same identity, but some asking for balance and some for revision.